buildpacks/builder.go

```diff
@@ -30,10 +30,17 @@ var Runtimes = map[string]string{
 
 // Build the Function at path.
 func (builder *Builder) Build(f faas.Function) (err error) {
-	// dervive the builder from the specificed runtime
-	packBuilder, ok := Runtimes[f.Runtime]
-	if !ok {
-		return errors.New(fmt.Sprint("unsupported runtime: ", f.Runtime))
+
+	// Use the builder found in the Function configuration file
+	// If one isn't found, use the defaults
+	var packBuilder string
+	if f.Builder != "" {
+		packBuilder = f.Builder
+	} else {
+		packBuilder = Runtimes[f.Runtime]
```

Comment:
`Runtimes` var name seems odd in this context, maybe it could be `Buildpacks` or `Runtime2Buildpack`, I know you haven't introduced that name.